compaction/compaction_manager.cc

```diff
@@ -198,9 +198,9 @@ std::vector<sstables::shared_sstable> in_strategy_sstables(compaction::table_sta
     }));
 }
 
-std::vector<sstables::shared_sstable> compaction_manager::get_candidates(const replica::table& t) {
+std::vector<sstables::shared_sstable> compaction_manager::get_candidates(compaction::table_state& t) {
```

Comment:
It's weird that table_state is in the compaction namespace.
Logically, it is an interface between the compaction_manager to table.
That's even more evident IMO when thinking about it as a compaction group.